src/components/publishing/imageset_preview.tsx

```diff
@@ -1,106 +1,131 @@
 import React, { Component } from "react"
 import styled from "styled-components"
 
-import IconImageset from "./icons/icon_imageset"
-
 import Fonts from "./fonts"
+import IconImageSet from "./icons/icon_imageset"
 
-const Wrapper = styled.div`
-  max-width: 580px;
-  width: 100%;
-  height: 150px;
+const FullWrapper = styled.div`
+  position: absolute;
+  bottom: 20px;
+  left: 20px;
+  min-height: 50px;
+  width: auto;
+  max-width: calc(100% - 80px);
   display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid white;
+  background: rgba(255, 255, 255, 0.8);
+  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.3);
+  padding: 20px;
+  `
+const MiniWrapper = styled.div`
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 10px 20px 10px 10px;
+  border: 1px solid #e5e5e5;
 `
-const Text = styled.div`
-  ${Fonts.avantgarde("s11")}
-  line-height: 1.35em;
-  margin: 0;
-`
-const Remaining = styled.div`
-  min-width: 50px;
-  padding: 10px;
+const MiniInner = styled.div`
+  display: flex;
+  justify-content: space-between;
   flex: 1;
+  width: auto;
+  margin-left: 20px;
+`
+const TitleWrapper = styled.div`
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  border: 1px solid #e5e5e5;
-  text-align: center;
+  justify-content: space-between;
+  height: 100%;
+  &[data-layout='mini'] {
```

Comment:
Like we talked about earlier today, it's definitely fine to use `data-layout` but if we're already using `this.props.layout` to make rendering decisions, it might make sense to let `TitleWrapper` accept the `layout` prop and use that to manipulate CSS. [Here](https://github.com/artsy/reaction/pull/199#discussion_r127554161) is the link to the helpful comment from @alloy.